src/hooks/useOffLicense.ts

```diff
@@ -14,7 +14,7 @@ function key(stowage: Stowage) {
     const { platform, arch } = stowage.host ?? host()
     rv = rv.join(`${platform}/${arch}`)
   }
-  let fn = `v${stowage.pkg.version}`
+  let fn = `v${stowage.pkg.version.raw ?? stowage.pkg.version}`
```

Comment:
k so… using version.raw means we use what was recording in versions.txt.

However I fear that `versions.txt` regenerates every time and uses `SemVer.toString()` rather than the raw version from the filename.

So this will break in prod when packages update.

I would like this because rendering calver properly is something I enjoy but I am unwilling to cause breakage.

Reply:
Praps `SemVer.toString()` needs calver preservation?